store/store.go

```diff
@@ -680,9 +680,11 @@ func (s *SnapUbuntuStoreRepository) Download(remoteSnap *snap.Info, pbar progres
 	if err != nil {
 		return "", err
 	}
-	s.setUbuntuStoreHeaders(req, "", remoteSnap.NeedsDevMode(), auther)
```

Comment:
this change loses us setting the User-Agent though, I would just do use the helper with "", false for now?